android/demo-app/src/main/java/com/stadiamaps/ferrostar/MainActivity.kt

```diff
@@ -152,7 +183,21 @@ class MainActivity : ComponentActivity(), AndroidTtsStatusListener {
             // but you can replace the styleURL with any valid MapLibre style URL.
             // See https://stadiamaps.github.io/ferrostar/vendors.html for some vendors.
             NavigationMapView(
-                styleUrl = "https://demotiles.maplibre.org/style.json", viewModel = viewModel)
+                styleUrl = "https://demotiles.maplibre.org/style.json", viewModel = viewModel) {
+                    uiState ->
+                  // Trivial, if silly example of how to add your own overlay layers.
+                  // (Also incidentally highlights the lag inherent in MapLibre location tracking
+                  // as-is.)
+                  Circle(
+                      center =
+                          LatLng(
+                              uiState.value.snappedLocation.coordinates.lat,
+                              uiState.value.snappedLocation.coordinates.lng),
+                      radius = 10f,
+                      color = "Blue",
+                      zIndex = 2,
+                  )
+                }
```

Comment:
Just leaving a comment for visibility to note that you can now (optionally) do a trailing closure composable to add your own MapLibre layers to the map ;)